src/qt/consolidateunspentwizardselectdestinationpage.cpp

```diff
@@ -0,0 +1,117 @@
+#include "consolidateunspentwizardselectdestinationpage.h"
+#include "ui_consolidateunspentwizardselectdestinationpage.h"
+
+#include "util.h"
+
+ConsolidateUnspentWizardSelectDestinationPage::ConsolidateUnspentWizardSelectDestinationPage(QWidget *parent) :
+    QWizardPage(parent),
+    ui(new Ui::ConsolidateUnspentWizardSelectDestinationPage)
+{
+    ui->setupUi(this);
+
+    ui->addressTableWidget->setSelectionMode(QAbstractItemView::SingleSelection);
+
+    // destination address selection
+    connect(ui->addressTableWidget, SIGNAL(itemSelectionChanged()), this, SLOT(addressSelectionChanged()));
+
+    ui->isCompleteCheckBox->hide();
```

Comment:
I actually make a comment about this. It was simpler to simply use a hidden field to me rather than write a full override. Line 22. We might as well make use of the capabilities already built in.... otherwise why don't we write everything from scratch....

    //This is used to control the disable/enable of the next button on this page.
    registerField("isCompleteSelectDestination*", ui->isCompleteCheckBox);

Reply:
I don't think it needs to be changed.

Reply:
Ok. I don't mind the checkbox. It was just a suggestion. The override would only need to `return QWizardPage::isComplete() && m_complete;`